operators/src/pro/cache/tile_cache.rs

```diff
@@ -77,14 +143,38 @@ pub enum CachedTiles {
     F64(Arc<Vec<RasterTile2D<f64>>>),
 }
 
+impl CachedTiles {
+    fn byte_size(&self) -> usize {
+        // TODO: include size of the Vec itself and the Arc
```

Comment:
it's just `std::mem::size_of::<CacheEntry>()`?

Reply:
you mean `CachedTiles`, right?

93cd66214d664374034b452067c33d5923694167